.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,190 @@
+        - Chrome (or Chromium-based)
+        - Safari
+        - Microsoft Edge
+        - Not applicable
+      default: 4
+
+  - type: checkboxes
+    id: install
+    validations:
+      required: true
+    attributes:
+      label:  |+
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)